functions/use_version.go

```diff
@@ -47,6 +47,18 @@ func downloadVersion(version string) error {
 		freebsd := strings.Trim(freebsdVersion[0], "\"")
 		url = fmt.Sprintf("https://github.com/gravitl/netclient/releases/download/%s/netclient-%s%s-%s", version, runtime.GOOS, freebsd, runtime.GOARCH)
 	}
+	if runtime.GOARCH == "arm" && runtime.GOOS == "linux" {
```

Comment:
is there a way we can use some Golang pkg to get this info, rather than running commands...

Reply:
Unfortunately,  no .  I checked the Go docs and also searched in Google.  There is not a direct method to get it.
There is one GOARM, but it's a compile option, sending to Go compiler. It's not a runtime variable.  There was someone who raised a request to add GOARM to runtime library. But the request was closed without any enhancement.